docs/guides/key-manager/get-controllers.md

```diff
@@ -0,0 +1,176 @@
+---
+sidebar_label: 'Get controller addresses'
+sidebar_position: 2
+---
+
+# Get controller addresses
+
+In this guide, you will learn how to:
+
+- retrieve all the addresses that have some permissions set on a Universal Profile,
+- retrieve the permission of each of these **controller addresses**.
+
+![Get controller addresses](/img/standards/lsp6/lsp6-address-permissions-array.jpeg)
+
+We will use the [_erc725.js_](../../tools/erc725js/getting-started.md) library to do this with just over 30 lines of code.
+
+## Setup
+
+```bash
+npm install erc725.js
+```
+
+## Step 1 - setup web3.js and erc725.js
+
+The first step is to setup both and _web3.js_ and _erc725.js_, connected to the LUKSO L14 network. We will also need the address of the Universal Profile that we want to get the controller addresses from.
+
+```js
+import { ERC725 } from '@erc725/erc725.js';
+import LSP6Schema from '@erc725/erc725.js/schemas/LSP6KeyManager.json';
+import Web3 from 'web3';
+
+// setup
+const myUniversalProfileAddress = '0xC26508178c4D7d3Ad43Dcb9F9bb1fab9ceeD58B5';
+const RPC_ENDPOINT = 'https://rpc.l16.lukso.network';
+
+// step 1 - setup erc725.js
+const web3 = new Web3(RPC_ENDPOINT);
+const erc725 = new ERC725(
+  LSP6Schema,
+  myUniversalProfileAddress,
+  web3.currentProvider,
+);
+```
+
+## Step 2 - Get the list of controller addresses
+
+The next step is to retrieve the list of controller addresses. In other words, addresses that have some permissions set and are allowed to interact with the Universal Profile.
+
+All the controller addresses are listed under the data key `AddressPermissions[]`. We can therefore obtain this list of addresses by:
+
+1. querying the `AddressPermissions[]` to get the array length, and know **how many controller addresses** have some permissions on the UP.
```

Comment:
```suggestion
1. query the `AddressPermissions[]` to get the array length and know **how many controller addresses** have some permissions on the UP.
```

Reply:
I'd like to keep `querying` and remove the comma.